templates/items/spell.hbs

```diff
@@ -104,6 +104,22 @@
                     </select>
                 </div>
             </div>
+            {{!-- Bound Spellcasting Origin --}}
+            {{#if document.parent}}
+            <div class="form-group input-select">
+                <label>{{ localize "DND5E.BoundSpellOrigin" }}</label>
+                <div class="form-fields">
+                    <select name="system.boundOrigin">
+                        {{#select system.boundOrigin}}
+                        <option value=""></option>
+                        {{#each document.availableSpellCastingOrigins as |scs|}}
+                        <option value="{{scs.value}}">{{scs.name}}</option>
+                        {{/each}}
+                        {{/select}}
+                    </select>
```

Comment:
Taking advantage of the suggested changes to `Actor5e#spellCastingClasses` allows us to simplify this:

```suggestion
                    <select name="system.sourceClass">
                        {{selectOptions document.parent.spellcastingClasses selected=system.sourceClass
                                        labelAttr="name" blank=""}}
                    </select>
```